docs/docsite/rst/plugins/inventory.rst

```diff
@@ -37,11 +37,11 @@ If the plugin is in a collection you need to use the fully qualified name:
 Using inventory plugins
 -----------------------
 
-To use an inventory plugin you need to provide an inventory source. Most of the time this is a file containing host information or a YAML configuration file with options for the plugin. You can use the ``-i`` flag to provide inventory sources or configure a default inventory path.
+To use an inventory plugin, you have to provide an inventory source. Most of the time this is a file containing host information or a YAML configuration file with options for the plugin. You can use the ``-i`` flag to provide inventory sources or configure a default inventory path.
```

Comment:
I agree with the spirit of this change - I think "must" is easier to translate. 
```suggestion
To use an inventory plugin, you must provide an inventory source. Most of the time this is a file containing host information or a YAML configuration file with options for the plugin. You can use the ``-i`` flag to provide inventory sources or configure a default inventory path.
```

Reply:
@acozine committed, thanks! will know

Reply:
ready_for_review